.github/workflows/push-images.yml

```diff
@@ -0,0 +1,32 @@
+name: "Push images to Docker Hub"
+on: workflow_dispatch
+jobs:
+  build:
+    name: "Build and push images"
+    runs-on: "ubuntu-24.04"
+    strategy:
+      fail-fast: false
+      matrix:
+        component:
+          - archivematica-dashboard
+          - archivematica-mcp-client
+          - archivematica-mcp-server
+    steps:
+      - name: "Check out repository"
+        uses: "actions/checkout@v4"
+      - name: "Set up buildx"
+        uses: "docker/setup-buildx-action@v3"
+      - name: "Login to Docker Hub"
```

Comment:
"Login" is a noun.

```suggestion
      - name: "Log in to Docker Hub"
```

Reply:
Thanks! Fixed in:
* AM https://github.com/artefactual/archivematica/pull/2070/commits/debcf6796715fbbceeaf18f979a3e4e3a787dcd1
* SS https://github.com/artefactual/archivematica-storage-service/pull/781/commits/0d8c0a1b58399afbc621069af30725a082286921